src/components/organisms/Settings/ProjectList/index.tsx

```diff
@@ -29,17 +33,73 @@ const ProjectList: React.FC<Props> = ({ teamId }) => {
     assetModalOpened,
     toggleAssetModal,
     onAssetSelect,
+    getMoreProjects,
   } = useHooks(teamId);
 
+  const projectLength = useMemo(
+    () => ({
+      currentProjectsLength: currentProjects.length || 0,
+      archivedProjectsLength: archivedProjects.length || 0,
+    }),
+    [currentProjects, archivedProjects],
+  );
+
+  const headers = {
+    current:
+      intl.formatMessage({
```

Comment:
I've asked @rot1024 whether we should do all new text with i18nnext or continue using react-intl. 
We will wait to see his response, but I am guessing we should change here to start using i18next.

Reply:
i18next is added here , please check

Reply:
Awesome. So that we can see the progress of migration, can you also remember to regenerate the legacy code with `yarn i18n:legacy`. This should remove the entries for the text you migrated!